src/clients/inst_autoinit.rb

```diff
@@ -86,6 +92,14 @@ def main
 
       Progress.Finish
 
+      # when installing from the online installation medium we need to
+      # register the system earlier because the medium does not contain any
+      # repositories, we need the repositories from the registration server
+      # TODO: maybe we will need it also in the autoupgrade case...
```

Comment:
we probably will need migration. Not sure if that method support it. We will see.